source/class/qx/theme/indigo/AppearanceDark.js

```diff
@@ -0,0 +1,231 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+   2004-2011 1&1 Internet AG, Germany, http://www.1und1.de
```

Comment:
guess you should replace this or at least add yours

Reply:
I don't have any copyright I would claim, so it's up to the group. I can remove it if you'd prefer?

Reply:
Not trying to be difficult, but I'm saying that I'm not claiming a copyright and don't want to. Does the Qooxdoo organization not have a standard copyright that can be used?

Reply:
please change the header to 
```
/* ************************************************************************

   qooxdoo - the new era of web development

   http://qooxdoo.org

   License:
     MIT: https://opensource.org/licenses/MIT
     See the LICENSE file in the project's top-level directory for details.

   Authors:
     * Scott Knick (@sknick)

************************************************************************ */
```

Reply:
Done.